_episodes/05-memory.md

```diff
@@ -7,8 +7,11 @@ questions:
 - "What is cognitive load and how does it affect learning?"
 - "How can we design instruction to work with, rather than against, memory constraints?"
 objectives:
-- "Identify when cognitive load is likely to limit learning."
+- "Remember the quantitative limit of human memory." 
+- "Identify the number of concepts required in a lesson by drawing a concept map." 
```

Comment:
Maybe "Apply a concept map to identify concepts and relationships required in a lesson"? That more accurately places this as an advanced-Blooms activity (arguably too advanced).